lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
It's to support a fallback in case SysV isn't activated for some reason. The C api will replace all the useful methods, like `#value`, `#value=`, `#increment`, `#reset`